### ISSUE TITLE
location of migrations folder?

### ISSUE THREAD

**Issue by tbowmo** [kriasoft/node-sqlite#63]
Documentation is not that clear for migrations. I've stumbled upon an error that I can't wrap my head around

I'm writing a node-red module in typescript, where I like to use the migrate feature, currently I have written it  like this (is Bluebird btw a pre-requisite? It's used in the examples):

```
        this.dbPromise = Promise.resolve()
            .then(() => open(this.file))
            .then((db) => db.migrate({}));
```

however, when I start nodered, which includes my module with the code above, I get the following error:

(node:17) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, scandir '/usr/src/node-red/migrations'

I have tried several locations for the migrations folder, but none seems to be right..

**Comment by UrGuardian4ngel** [comment]
By default, everything inside of the `./migrations/` folder.
But [looking through the code](https://github.com/kriasoft/node-sqlite/blob/5d092c5844fa7a096d1539da91713232152cde5a/src/Database.js#L148-L150), it seems like the path is configurable by passing a `migrationsPath` property to the `migrate(options)` method.

In my application, that folder is at **root level** (just like in this repository).
But running it from an NPM module, I still was using the one in the project root (same level of _node_modules_).

**TL;DR:** I'm guessing that by default, it is the `migrations/` folder, resolved from the path of the entry point script...

**Comment by theogravity** [comment]
The v4 beta now has the migrations folder set to the project root as default, which can also be customized.
